ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/event-handlers.j2

```diff
@@ -53,19 +56,18 @@ event-handler {{ handler.name }}
       regex {{ handler.trigger_on_logging.regex }}
 {%                 endif %}
 {%             elif handler.trigger is arista.avd.defined("on-intf") and handler.trigger_on_intf.interface is arista.avd.defined %}
-{%                 if handler.trigger_on_intf.ip is arista.avd.defined(true) or
-                     handler.trigger_on_intf.ipv6 is arista.avd.defined(true) or
-                     handler.trigger_on_intf.operstatus is arista.avd.defined(true) %}
-{%                     set trigger_on_intf_cli = "trigger on-intf " ~ handler.trigger_on_intf.interface %}
-{%                     if handler.trigger_on_intf.operstatus is arista.avd.defined(true) %}
-{%                         set trigger_on_intf_cli = trigger_on_intf_cli ~ " operstatus" %}
-{%                     endif %}
-{%                     if handler.trigger_on_intf.ip is arista.avd.defined(true) %}
-{%                         set trigger_on_intf_cli = trigger_on_intf_cli ~ " ip" %}
-{%                     endif %}
-{%                     if handler.trigger_on_intf.ipv6 is arista.avd.defined(true) %}
-{%                         set trigger_on_intf_cli = trigger_on_intf_cli ~ " ip6" %}
+{%                 set trigger = [] %}
+{%                 for intf_change in ['operstatus', 'ip', 'ipv6'] %}
```

Comment:
cant we have ip6 instead of ipv6

Reply:
We need to keep the correct key_name 'ipv6' but EOS supports `ip6`.